### PR TITLE
Rebase 20230801

### PR DESCRIPTION
**Pull request by guplersaxanoid**
# Description
Sync with main SDK

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] I have tested locally
- [ ] I have performed a self review of my changes
- [ ] Updated any relevant documentation
- [ ] Linked to any relevant issues
- [ ] I have added tests relevant to my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] My code is up to date with the base branch
- [ ] I have updated relevant changelogs. [We suggest using chan](https://github.com/geut/chan/tree/main/packages/chan)

### CODE REVIEW

File: packages/common-near/test/project_1.0.0_node_options.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+specVersion: 1.0.0
```
Comment (jiqiang90) [inline packages/common-near/test/project_1.0.0_node_options.yaml]: Is this file used anywhere?

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        description: 'Benchmark time(h/m/s)'
+        default: 30m
+        require: true
+      indexerVersion:
+        description: 'indexer version'
+        default: 'latest'
+        require: true
+      deployment:
+        description: 'Deployment'
+        default: 'QmakYyTWzTxaDU842CczyDWV51Ay9E55k7kbRRXTB3f395'
```
Comment (jiqiang90) [inline .github/workflows/benchmark.yml]: need to a near project, and also near endpoint

File: .github/workflows/scripts/benchmark/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ENV DB_PASS=postgres
+ENV DB_DATABASE=postgres
+ENV DB_HOST=postgres
+ENV DB_PORT=5432
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Install app dependencies
+RUN npm i @subql/node@$INDEXER_VERSION -g
```
Comment (jiqiang90) [inline .github/workflows/scripts/benchmark/Dockerfile]: this also need to be subql/node-near

File: .github/workflows/benchmark.yml (hunk 2 above)
Comment (guplersaxanoid) [reply .github/workflows/benchmark.yml]: should I deploy a starter project?

**Review by stwiname — APPROVED** [review]

**Pull request merged**